src/content/blog/projects.mdx

```diff
@@ -0,0 +1,136 @@
+
+<figure>
+  <Image src={DateMe} alt="Val Example" />
+  <figcaption>
+    [Date Me Directory](https://val.town/v/stevekrouse/dateme) - an alternative
+    dating website that became an unwieldy collection of vals with many
+    disorganized forks
+  </figcaption>
+</figure>
+
+Instead most users either stuffed all their code into a single super long val, just made all their changes directly to their main production vals (sometimes breaking it briefly), or moved off Val Town entirely. Clearly we need a solution to dramatically increase the scope of what you can build on Val Town.
+
+## Enter Projects
+
+Projects are containers for multiple vals, files, and folders that are versioned, branched, forked, and merged back collectively. With Projects, you can create feature branches and test changes without affecting production.
+
+As you may know, Val Town deploys your code in 100ms. This is a magical way to code _directly in the cloud_ – [instead of on localhost and deploying to the cloud later](https://dx.tips/the-end-of-localhost). This kind of live feedback loop is incredibly fun. It also means that you are building in the exact same environment that you'll deploy to, cutting out all the complexity of managing multiple environments.
+
+However, as described above, just because you're deploying to the cloud on every save doesn't mean you want to deploy to _production_ on every save. Feature branches in projects give you the best of both worlds: you can rapidly prototype in the exact same environment that you'll deploy to, but without affecting production. It's like localhost and PR previews had a baby: super fast iteration cycles, paired with hosted URLs you can immediately share.
+
+And the final kicker is scale. Our vision is that once your app is running on Val Town, it's deployed. You don't have to worry about provisioning more or less compute. Our serverless platform will automatically scale your app to handle any number of users. (Our Scale Plan is slated to launch in Q2 2025. For now, Val Town scales to 100k and 1m runs per day on [the Free and Pro plans](https://www.val.town/pricing), respectively.)
+
+## Projects are not git repos
+
+We considered building Projects as git repos. We decided that becoming a git host at scale was too much complexity for the small subset of git's features that we wanted (diffs, branches, merges). We realized that it'd be much simpler to build our own version control system.
+
+This sounds like sacrilege – even to us. Everyone at Val Town is aa happy git and GitHub user. Yet if you think about it, many any of the web's pro tools had to build their own version control, including Google Docs, Figma, and Notion. We feel very heartened to read about how speculative and uncertain this decision felt at Figma to build their own multiplayer system instead of using OTs (operational transforms), which was the standard at the time given its success in Google Docs.
```

Comment:
```suggestion
This sounds like sacrilege – even to us. Everyone at Val Town is a happy git and GitHub user. Yet if you think about it, many any of the web's pro tools had to build their own version control, including Google Docs, Figma, and Notion. We feel very heartened to read about how speculative and uncertain this decision felt at Figma to build their own multiplayer system instead of using OTs (operational transforms), which was the standard at the time given its success in Google Docs.
```